.goreleaser.yml

```diff
@@ -1,22 +1,23 @@
 builds:
-  - main: ./cmd/sqs/main.go
+  - main: ./cmd/connector/main.go
```

Comment:
This is good and actually preferred, but it doesn't match reality. Can you rename the folder `cmd/sqs` to `cmd/connector`? :pray: And of course change anything that references `cmd/sqs` to `cmd/connector`.